src/process/index.js

```diff
@@ -32,30 +64,96 @@ export default class ProcessServer {
+      if (_path.includes('systemd')) continue;
+      const cwdPath = _cwdPath.toLowerCase().replaceAll('\\', '/');
       const path = _path.toLowerCase().replaceAll('\\', '/');
+      if (path.startsWith('c:/windows')) continue // system processes (wine)
+      if (_path.includes('webhelper')) continue; // CEF Processes
+      // TODO: add 'dolphin-emu' to database for linux executable
+      if (_path.endsWith('/bin/dolphin')) continue; // KDE file manager, not Dolphin Emulator
       const toCompare = [];
-      const splitPath = path.split('/');
-      for (let i = 1; i < splitPath.length; i++) {
-        toCompare.push(splitPath.slice(-i).join('/'));
+      let newPath
+      if (path.includes(' --')) {
+        newPath = path.split(' --')[0];
       }
+      else
+      {
+        newPath = path;
+      }
+      newPath = newPath.substr(newPath.lastIndexOf('/') + 1);
+
+      // log(`performance checkpoint: ${(performance.now() - startTime).toFixed(2)}ms`);
 
-      for (const p of toCompare.slice()) { // add more possible tweaked paths for less false negatives
-        toCompare.push(p.replace('64', '')); // remove 64bit identifiers-ish
-        toCompare.push(p.replace('.x64', ''));
-        toCompare.push(p.replace('x64', ''));
-        toCompare.push(p.replace('_64', ''));
+      toCompare.push(newPath);
+      if (path.includes('.exe')) {
+        const part2 = path.split('/').slice(-2).join('/');
+        toCompare.push(part2);
+        replaceAll(toCompare, bitness_suffixes);
       }
 
+      // TODO: Convert into an inline function similar to findInObjArray
+      // TODO: Don't try to match the running executable more than once
       for (const { executables, id, name } of DetectableDB) {
-        if (executables?.some(x => {
-          if (x.is_launcher) return false;
-          if (x.name[0] === '>' ? x.name.substring(1) !== toCompare[0] : !toCompare.some(y => x.name === y)) return false;
-          if (args && x.arguments) return args.join(" ").indexOf(x.arguments) > -1;
-          return true;
-        })) {
+        if (
+          executables?.some((known_exe) => {
+            if (known_exe.is_launcher) return false;
+            if (known_exe.name[0] === '>') {
+              if (known_exe.name.substring(1) === toCompare[0]) {
+                // TODO: Deduplicate with that version at the end of the following 'else' statement
+                if (args && known_exe.arguments) {
+                  // log(`Match Level 1: "${name}" via ${known_exe.name} <==> ${running}`);
+                  return args.join(" ").indexOf(known_exe.arguments) > -1;
+                }
+              }
+            } else {
+              if (
+                toCompare.some((running) => {
+                  // explicit match first
+                  if (known_exe.name === running) {
+                    // log(`Match Level 2: "${name}" via ${known_exe.name} <==> ${running}`)
```

Comment:
These log comments aren't really necessary to understand what's going on but if here's two options I can suggest:

Option 1: make it `console.debug` instead of `.log`, so we can configure logging levels and skip `debug` by default. This gives you an easy toggle for traces if you need them, and not just a comment.

```suggestion
                    // debug(`Match Level 2: "${name}" via ${known_exe.name} <==> ${running}`)
```

or

```suggestion
```